ocrd_network/ocrd_network/deployer.py

```diff
@@ -281,11 +280,7 @@ def start_native_processor(self, client: SSHClient, processor_name: str, queue_u
         self.log.info(f'Starting native processor: {processor_name}')
         channel = client.invoke_shell()
         stdin, stdout = channel.makefile('wb'), channel.makefile('rb')
-        if is_bashlib_processor(processor_name):
-            cmd = f'ocrd processing-worker {processor_name} --database {database_url} ' \
-                f'--queue {queue_url}'
-        else:
-            cmd = f'{processor_name} --database {database_url} --queue {queue_url}'
+        cmd = f'{processor_name} --database {database_url} --queue {queue_url}'
```

Comment:
Nothing fancy here at all. Just a `system(3)` call in the end, only via bash instead of Python.

Kitodo, I guess you mean our [OCR-D Manager scripts](https://github.com/slub/ocrd_manager)?

You can do almost anything in bash BTW, just the data types are poor. (It does have are arrays and associative arrays now, but you cannot nest them.) And for string processing, you often find yourself calling external tools like `sed`, `awk`, `cut` etc.

Reply:
> Kitodo, I guess you mean our [OCR-D Manager scripts](https://github.com/slub/ocrd_manager)?

Yes.

Reply:
> was positively amazed when I saw almost everything is done in bash in Kitodo.

Well, we started out thinking Manager would not have to do much, some data transfer and `ocrd workspace` tasks. But it quickly turned out there's a lot more to it. We already thought about migrating to Python, but still some things (like CoW transfers, rsync and SSH) can only be on the shell anyway (and doing lots of callouts from Python would make it more complex than just doing everything in bash IMO).